implementations/rust/ockam/ockam_app/src/invitations/commands.rs

```diff
@@ -210,9 +213,9 @@ async fn refresh_inlets<R: Runtime>(
                     .stdout_capture()
                     .run();
                     match create_inlet(&i).await {
-                        Ok(inlet_socket_addr) => {
-                            inlets_socket_addrs
-                                .push((invitation.invitation.id.clone(), inlet_socket_addr));
+                        Ok(socket_addr) => {
```

Comment:
I think that there are plenty of opportunities to refactor that code and the code on line 234 by having better entities. Maybe having a direct representation called `TheirService` which would contain invitation information (who? what?) and inlet data (socket address). 

And having `Services` instead of `InvitationState` where there's an interface with a method like `service_invitation_accepted(invitation_id: String,  created_inlet: InletStatus)` which would update the internal data: remove the received invitation and create a new service inside a `their_services` field.

What I would be aiming for would be a better separation of interaction with the external world and updates to the internal state in a more encapsulated way (right now we expose bare maps).

In terms of vocabulary we could have:

 - `SentInvitation { recipient_email: String, service_outlet_name: String....}`
 - `ReceivedInvitation { sender_email: String, ....}`
 - `ServiceOutlet { name: String,  , socket_addr: SocketAddr, node_name: String, connected: bool }` 
 - `ServiceInlet { name: String, sender_email: String, socket_addr: SocketAddr, node_name: String, connected: bool }`